### PR TITLE
Refactor staking options to be base / quote instead of project / usdc

### PR DESCRIPTION
**Pull request by brittcyr**

### CODE REVIEW

**Comment by brittcyr** [comment]
Nothing really interesting in here, but just take a look since this is a pretty large change. Make sure base/quote is the right terminology for this

File: programs/staking-options/src/instructions/issue.rs [hunk 1]
```diff
@@ -2,11 +2,12 @@ use anchor_spl::token::{Mint, Token, TokenAccount};
 
 pub use crate::*;
 
-pub fn issue(ctx: Context<Issue>, amount: u64) -> Result<()> {
+pub fn issue(ctx: Context<Issue>, amount: u64, strike: u64) -> Result<()> {
```
Comment (DonDuala) [inline programs/staking-options/src/instructions/issue.rs:5]: What does adding Strike do?

File: tests/staking-options.ts [hunk 2]
```diff
@@ -367,25 +370,25 @@ describe('staking-options', () => {
...
+          baseTokenMint.toBuffer(),
         ],
         program.programId,
       ));
 
     // Wait for the old one to expire.
     await new Promise((r) => setTimeout(r, 100_000));
 
-    optionExpiration = Math.floor(Date.now() / 1000 + 1000);
+    optionExpiration = Math.floor(Date.now() / 1000 + 2000);
```
[the remark this answers is not in the document]
Comment (brittcyr) [reply tests/staking-options.ts:391]: This makes the test less flaky if there is clock drift between local and the chain.

File: programs/staking-options/src/common.rs [hunk 3]
```diff
@@ -29,15 +29,18 @@ pub struct State {
...
     // Mint of the reward token
-    pub project_token_mint: Pubkey,
+    pub base_token_mint: Pubkey,
 
     // The account that will receive payments on the options.
-    pub usdc_account: Pubkey,
+    pub quote_account: Pubkey,
 
-    // Vector of all strikes for an SO. Limit 100. For monitoring only. A strike is number of USDC atoms per full project token.
+    // Vector of all strikes for an SO. Limit 100. For monitoring only. A strike is number of USDC atoms per full base token.
```
[the remark this answers is not in the document]
Comment (brittcyr) [reply programs/staking-options/src/common.rs]: fixed

**Pull request merged**